### ISSUE TITLE
Continue loading FHIR data after partial failure

### ISSUE THREAD

**Issue by savanahmueller** [chronic-care/mcc-project#292]
Complete all FHIR queries and display available data, even when one or more errors are returned, but others succeed.

![Screenshot 2023-06-02 102738.png](https://images.zenhubusercontent.com/61b8c8b30aba1114f9de2b8d/329c21d8-1aea-477c-8f89-1325b7dfd2fb)

Using the Nancy Smart login from Cerner, the load times out and provides this response. 

One successful query is enough.

**Comment by drdavec** [comment]
The browser console for above errors shows a CORS error, but only for Immunization query.  Assume this is an error on Cerner's server config?  In general, if some of the queries succeed, we should display those data while reporting errors from others.

We don't know if the remaining Cerner queries for nancysmart will be successful because app aborts after this one failure.

**Comment by drbgfc** [comment]
All resources are now handled in the sense that the application can continue loading and display a warning if any resource fails. The changes were extensive enough that they require decent testing as every resource was touched and most of them were modularized. This means there could be a mistake (hopefully not, I was careful) but also that we need tests to confirm it works for each resource having an issue and various combinations of them having issues in different orders. And, we need to verify the display of each relevant resource in the relevant tabs.
In addition, I added back support for surveyResults, previously, this was removed from the source code because it had a 400 error. However, with this new non-terminating load feature we can still load it in cases when it works and load other data when it does not. I can remove this resource if you would prefer.
Note: If we load nancysmart we can see  an example of 2 warnings caught and displayed, (one for immunizations and one for Obs Survey).
As for recording provenance for conditions, I didn't record it as that's the way the code read prior. Just let me know what's needed.
9:03
(pushed to the develop branch)
